### ISSUE TITLE
Rules doc structure

### ISSUE THREAD

**Issue by kevinymfan** [chairleague/hots-rules#29]
Currently, the rules doc has a lot of things that seem somewhat counterintuitive and may be confusing for someone looking for specific information quickly. Some examples:

- The "Tournaments" section has a "Divisions" subsection even though there's also a "Divisions" main section

- "Team Timezones" subsection is found under "League Play" section rather than "Teams" section

- "Substitutions" subsection is found under "League Play" section instead of "Matches" section

- "Season Rating" subsection is under "Divisions" section instead of "Teams" section

May I go ahead and create a pull request restructuring the rules doc?

**Comment by Durgulach** [comment]
I dont want to speak out of turn, but I know Jova is the only one who can actually make changes live so I dont see any reason why you shouldnt make the pull request with the restructuring and he can pick and choose what he wants to incorporate into the live version.

**Comment by Hammwich** [comment]
I agree with the proposal, getting stuff in order if anything is just a nice QOL improvement.

**Comment by superjova** [comment]
Yea I'm down. We can reorganize in a pull request. 

I'm no English major that's for sure
